console/console-init/ui/src/modules/address/containers/AddressDefinitionContainer/AddressDefinitionContainer.tsx

```diff
@@ -136,6 +149,62 @@ export const AddressDefinitionContainer: React.FunctionComponent<IAddressDefinit
           setTopicForSubscripitons(topics);
         }
       }
+      if (
+        type.toLowerCase() === "subscription" ||
+        type.toLowerCase() === "queue"
+      ) {
+        const deadletter_addresses = await client.query<IAddressResponse>({
+          query: RETURN_DLQ_ADDRESSES_FOR_SUBSCRIPTION_AND_QUEUE(
+            addressspaceName,
+            namespace,
+            type
+          )
+        });
+        if (
+          deadletter_addresses.data &&
+          deadletter_addresses.data.addresses &&
+          deadletter_addresses.data.addresses.addresses.length > 0
+        ) {
+          const deadletters = deadletter_addresses.data.addresses.addresses.map(
+            address => {
+              return {
+                key: address.spec.address,
+                value: address.spec.address,
+                label: address.metadata.name
+              };
+            }
+          );
+          setDeadlettersOptions(deadletters);
+        }
+      }
+      if (
```

Comment:
no need to fetch the data twice. we have to use the same addresses i.e deadLetter addresses for both dropdown